samples/excel/excel-library/overview/wwwroot/index.html

```diff
@@ -0,0 +1,37 @@
+﻿<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+
+    <title>Samples | IgniteUI for Blazor | Infragistics</title>
+    <meta name="description" content="Explore samples of IgniteUI for Blazor components.">
+    <meta name="keywords" content="SampleName, Examples, Blazor, IgniteUI for Blazor, Components, Infragistics">
+    <meta name="author" content="Infragistics">
+
+    <base href="/" />
+    <link href="index.css" rel="stylesheet" />
+
+    <link href="https://static.infragistics.com/xplatform/images/browsers/blazor.ico"  rel="icon" type="image/x-icon">
+</head>
+
+<body>
+    <app class="app"><img class="app-loading" src="https://static.infragistics.com/xplatform/images/browsers/loading.gif" /></app>
+    <!-- <app>Loading...  </app> -->
+
+    <div class="app-error-ui">
+        An unhandled error has occurred.
+        <a href="" class="reload">Reload</a>
+        <a class="app-error-dismiss">ðŸ—™</a>
+    </div>
+    <!-- importing blazor bundle for IG Excel: -->
+    <script src="_framework/blazor.webassembly.js"></script>
+    <script src="_content/IgniteUI.Blazor.Documents.excel/excel.js"></script>
```

Comment:
you need to add this line to [index.html](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/browser/IgBlazorSamples.Client/wwwroot/index.html#L32)